cloudify_hostpool/storage/sqlite.py

```diff
@@ -0,0 +1,123 @@
+# #######
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+import json
+import sqlite3
+from base import AbstractStorage
```

Comment:
we prefer using absolute imports.

i.e : from cloudify_hostpool.storage.base import AbstractStorage
also, imports should be separated by a line break between logical segments.
First are the built-in python libraries (like you did), next are third party imports, and last are imports from the current project.